waspc/examples/todoApp/src/client/Todo.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { test, expect } from 'vitest'
+import { screen } from '@testing-library/react'
+
+import { mockQuery, renderWrapped } from '@wasp/vitest.helpers'
```

Comment:
This probably needs a better name than `vitest.helpers`. Any ideas?

Reply:
I would go with `@wasp/test` for now?

Reply:
Maybe put everything related to testing in a `testing` folder.

```c
web-app/
-- testing/
---- vitest.setup.ts
---- index.ts // or helpers.ts
```